app/views/shared/_recently_updated.html.erb

```diff
@@ -1,23 +1,20 @@
-<% extra_class ||= '' %>
-
-<section id="recently-updated" class="recently-updated <%= extra_class %>">
-  <div class="content">
-    <header>
-      <h1 class="label"><%= t('latest_feed.title') %></h1>
-    </header>
-    <% if recently_updated.any? %>
-      <%= render partial: defined?(documents_source) && documents_source == 'rummager' ?
-                "shared/recently_updated_documents_from_rummager" :
-                "shared/recently_updated_documents",
-                locals: { recently_updated: recently_updated } %>
-    <% else %>
-      <p><%= t('latest_feed.no_updates') %></p>
-    <% end %>
-    <%= render 'shared/feeds', atom_url: atom_url %>
-    <% if local_assigns[:see_all_link] %>
-      <p class="see-all" <%= t_lang('document.see_all') %> >
-        <%= link_to t('document.see_all'), see_all_link, class: 'govuk-link' %>
-      </p>
-    <% end %>
-  </div>
+<section id="recently-updated">
+  <%= render "govuk_publishing_components/components/heading", {
+    text: t('latest_feed.title'),
+  } %>
+  <% if recently_updated.any? %>
+    <%= render partial: defined?(documents_source) && documents_source == 'rummager' ?
+      "shared/recently_updated_documents_from_rummager" :
+      "shared/recently_updated_documents",
+      locals: { recently_updated: recently_updated } 
+    %>
+  <% else %>
+    <p><%= t('latest_feed.no_updates') %></p>
+  <% end %>
+  <% if local_assigns[:see_all_link] %>
+    <p class="govuk-!-margin-top-3 govuk-!-margin-bottom-6" <%= t_lang('document.see_all') %> >
```

Comment:
`govuk-body` again? I also feel a bit sus about these override classes here.

Reply:
>I also feel a bit sus about these override classes here.

How come? I thought they are preferred approach to avoid extra CSS

![image](https://user-images.githubusercontent.com/71266765/126480283-4f37512c-2046-4963-bf72-5cc48f316ad2.png)

Reply:
Basically, I don't really understand why we need it. The space difference feels negligible from a design perspective and muddies the convention of consistent spacing across govuk.

Reply:
> The space difference feels negligible from a design perspective

It's subtle enough, removed.